sdk/core/core-http/rollup.config.ts

```diff
@@ -68,7 +67,8 @@ const browserConfig = {
   },
   plugins: [
     nodeResolve({
-      mainFields: ["module", "main", "browser"]
+      mainFields: ["module", "browser"],
+      preferBuiltins: false
```

Comment:
Sorry if this was already commented on (it looks like most comments have disappeared for me) but why was this change needed again? The tests have been passing (I think!) so was curious what issue triggered this change.

Reply:
Also `main` disappeared

Reply:
@ramya0820 Can you respond to @chradek's comment above on the need to add `preferBuiltins: false`?

Reply:
This was pulled in from master - Eng Sys had updated all the libraries to make sure rollup works as expected where explicable. Can refer to #3326 for the issue and #4613 for the changes in master

Reply:
About comments disappearing, I think it's because the changes have been commented on were reverted and are not applicable to PR anymore.